_ko/tour/tuples.md

```diff
@@ -5,13 +5,79 @@ title: Tuples
 discourse: true
 
 partof: scala-tour
-num: 
+
+num: 6
 language: ko
 
 next-page: mixin-class-composition
 previous-page: traits
+topics: tuples
 
+redirect_from: "/tutorials/tour/tuples.html"
 ---
 
-(this section of the tour has not been translated yet. pull request
-with translation welcome!)
\ No newline at end of file
+스칼라에서 하나의 튜플은 정해진 갯수의 각각 구별된 타입을 갖는 엘리먼트들을 포함하는 하나의 값이다. 튜플은 불변적이다.
```

Comment:
It's a good idea to translate people so that you can understand them better. And I translate as much as possible into Korean. I prefer a way to increase readability in parentheses if necessary.

스칼라에서 하나의 튜플은 정해진 갯수의 각각 구별된 타입을 갖는 엘리먼트들을 포함하는 하나의 값이다.

How about this?

스칼라에서 튜플은 정해진 요소(element)를 가지는 값으로 각 요소는 고유한 타입을 가집니다.